### PR TITLE
Multiple fixes

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
- Fix Openpose service
- Fix Torso in Gazebo
- Fix navigation costmaps

Please keep the branch after merging.

### CODE REVIEW

File: scripts/openpose.service [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=image_recognition_openpose node
+
+[Install]
+WantedBy=multi-user.target
+
+[Service]
+User=amigo
+Restart=on-abort
+ExecStart=/bin/bash -c 'source /home/amigo/.tue/setup.bash && export PATH=/usr/local/cuda/bin:$PATH && export LD_LIBRARY_PATH=/usr/local/cuda/lib64$:$LD_LIBRARY_PATH && rosrun image_recognition_openpose openpose_node _net_input_width:=368 _net_input_height:=368 _net_output_width:=368 _net_output_height:=368 _model_folder:=/home/amigo/openpose/models/ __ns:=hero/openpose image:=/hero/head_rgbd_sensor/rgb/image_raw'
```
Comment (PetervDooren) [inline scripts/openpose.service]: Is there a way to make this more readable?

File: parameters/navigation/global_costmap_gmapping.yaml [hunk 2]
```diff
@@ -5,6 +5,7 @@ robot_base_frame: /base_footprint
 rolling_window: false
 track_unknown_space: false
 resolution: 0.05
+robot_radius: 0.24
```
Comment (PetervDooren) [inline parameters/navigation/global_costmap_gmapping.yaml]: I assume this was measured. Or was it a guestimate

Comment (MatthijsBurgh) [reply parameters/navigation/global_costmap_gmapping.yaml]: Measured and compared to values used by you and @JosjaG

File: scripts/openpose.service (hunk 1 above)
Comment (MatthijsBurgh) [reply scripts/openpose.service]: I don’t think so. Maybe multi-line would be possible.

Comment (PetervDooren) [reply scripts/openpose.service]: multi line is possible using a backslash \

Comment (MatthijsBurgh) [reply scripts/openpose.service]: Indeed, testing is required

Comment (MatthijsBurgh) [reply scripts/openpose.service]: Works

**Pull request merged**